node/actors/network/src/rpc/push_signature.rs

```diff
@@ -0,0 +1,42 @@
+//! Defines RPC for passing consensus messages.
+use std::sync::Arc;
+
+use crate::{mux, proto::gossip as proto};
+use anyhow::Context;
+use zksync_consensus_roles::attester::{self, L1Batch};
+use zksync_protobuf::ProtoFmt;
+
+/// Signature RPC.
+pub(crate) struct Rpc;
+
+impl super::Rpc for Rpc {
+    const CAPABILITY_ID: mux::CapabilityId = 5;
```

Comment:
add it to test_capability_rpc_correspondence test - it currently acts as a register for all RPCs to prevent reuse of CAPABILITY_IDs